src/content/protocols/morpho.md

```diff
@@ -0,0 +1,200 @@
+---
+protocol: "Morpho"
+website: "https://morpho.org/"
+x: "https://x.com/MorphoLabs"
+github: ["https://github.com/morpho-org"]
+defillama_slug: ["morpho-blue"]
+chain: "Ethereum"
+stage: 1
+reasons: []
+risks: ["L", "M", "M", "M", "L"]
+author: ["mmilien_"]
+submission_date: "2024-11-27"
+publish_date: "2025-05-21"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Morpho is a lending protocol with permissionless market creation. It enables the deployment of minimal and isolated lending markets by specifying: one collateral asset, one loan asset, a Liquidation Loan To Value (LLTV), an Interest Rate Model (IRM), and an oracle. Users may lend funds directly on individual Morpho Markets or through Morpho Vaults. These vaults are created permissionlessly by third parties, or risk curators, and offer managed lending strategies by aggregating different Morpho Markets. Morpho governance operates within a very limited scope and without direct control over the Morpho protocol.
+
+# Ratings
+
+## Chain
+
+Morpho is deployed on different chains. This review is based on the Ethereum mainnet deployment.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Morpho (markets) protocol and Morpho Vaults are non-upgradeable. No permissions exist in the Morpho protocol that could affect users' funds and unclaimed yield or could otherwise result in non-expected protocol performance. Permissions in Morpho Vaults are owned by the vault creators themselves, aka _Curators_, and thus are not centralized under Morpho governance.
```

Comment:
```suggestion
The Morpho (markets) protocol and Morpho Vaults are non-upgradeable. No permissions exist in the Morpho protocol that could affect _users' funds_ and _unclaimed yield_ or could otherwise result in non-expected _protocol performance_. Permissions in Morpho Vaults are owned by the vault creators themselves, aka _Curators_, and thus are not centralized under Morpho governance.
```